packages/client/src/processes/routes/routes.tsx

```diff
@@ -1,7 +1,10 @@
 import { RouteObject } from 'react-router-dom'
-
 import { routesPath } from './routesPath'
+import Login from 'pages/login/ui'
```

Comment:
Можно сократить путь до 
```suggestion
import Login from 'pages/login'
```

Reply:
Поправил